admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -711,12 +711,6 @@
   }
 
   #product-images-dropzone.dropzone {
-    display: flex;
-    align-items: flex-start;
-    justify-content: space-between;
-    border-radius: 4px;
```

Comment:
I guess the border-radius removal is a miss

Reply:
Isn't it the global border of the dropzone component that we can see there? 
<img width="1332" alt="image" src="https://user-images.githubusercontent.com/14963751/194336238-408ee90a-9a38-4184-8e60-f59a0999becb.png">

Reply:
its this one, maybe it just not done correctly but no it has no impact.
![image](https://user-images.githubusercontent.com/51944825/195290905-72e7340b-c710-408d-9e7a-100896ce08cd.png)

Reply:
Alright, @margud thanks for the investigation!

I would suggest to move it to the div upper then, it would make sense from a design point of view :)

Then it's a 🍬 from me!

Reply:
Sure i can do that but all the fields bellow like summary and description doesn't have the border radius, so is it really good idea to have this specific radius just for image div? I think that looks strange. @NeOMakinG 
![image](https://user-images.githubusercontent.com/51944825/195551976-e65ed3f2-65d8-47ba-aa34-6f7fc95e2755.png)

Reply:
> Sure i can do that but all the fields bellow like summary and description doesn't have the border radius, so is it really good idea to have this specific radius just for image div? I think that looks strange. @NeOMakinG ![image](https://user-images.githubusercontent.com/51944825/195551976-e65ed3f2-65d8-47ba-aa34-6f7fc95e2755.png)

Meh, I was thinking they had, in that case, let's go 👍 

Thanks for the explaining